.eslintrc.json

```diff
@@ -57,6 +57,7 @@
         "$": false,
 
         "window": false,
+        "document": false,
```

Comment:
Sorry for mixed signals, but please revert this.
`document` is a generic word and in Brackets usually refer to the Document object.
I think you can go with jquery here, can't you? `$("#theme-settings-done-btn")`